src/seedu/addressbook/data/person/Name.java

```diff
@@ -36,6 +36,23 @@ public static boolean isValidName(String test) {
         return test.matches(NAME_VALIDATION_REGEX);
     }
 
+
+    /**
+     * @param name
+     * @return A String with first letter of every word capitalised
+     */
+    private static String capitaliseName(String name)  {
```

Comment:
You can make the method name more generic; this is not specific to address book.